scripts/rebuilder.py

```diff
@@ -27,6 +34,88 @@ def is_tier0_package(pkg: str) -> bool:
     ]
 
 
+def filter_unsupported_pkgs(pkgs: set[str]) -> set[str]:
+    """Filters out unsupported packages and returns the rest.
+
+    Args:
+        pkgs (set[str]): List of package names
+
+    Returns:
+        set[str]: List of package names without unsupported packages
+
+    """
+    unsupported_pkgs = {"dotnet6.0", "dotnet7.0"}
+    for p in unsupported_pkgs:
+        pkgs.discard(p)
+    return pkgs
+
+
+# Packages in CentOS Stream that are built by clang
+def get_tier1_pkgs(version: int) -> set[str]:
+    base = dnf.Base()
+    conf = base.conf
+    for c in "AppStream", "BaseOS", "CRB":
+        base.repos.add_new_repo(
+            f"{c}-{version}-source",
+            conf,
+            baseurl=[
+                f"https://mirror.stream.centos.org/{version}-stream/{c}/source/tree/"
+            ],
+        )
+    repos = base.repos.get_matching("*")
+    repos.disable()
+    repos = base.repos.get_matching("*-source*")
+    repos.enable()
+
+    base.fill_sack()
+    q = base.sack.query(flags=hawkey.IGNORE_MODULAR_EXCLUDES)
+    q = q.available()
+    q = q.filter(requires=["clang"])
```

Comment:
Does are `require` key really identifies the dependency at build-time as the documentation of this function suggests? `# Packages in CentOS Stream that are built by clang`

Shouldn't this be search for `biuld_requires`?

Reply:
There is no option for build_requires as far as I  know.  This works because it is querying a sources repo.